ui/src/app/cohort-review/detail-all-events/detail-all-events.component.html

```diff
@@ -18,8 +18,7 @@
     </clr-dg-filter>
   </clr-dg-column>
 
-  <clr-dg-row *clrDgItems="let datum of data"
-    (click)="onClick(datum)">
```

Comment:
Why was this removed? Just out of curiosity/not understanding.

Reply:
The downstream event was not actually used.